src/code/client.coffee

```diff
@@ -548,15 +548,23 @@ class CloudFileManagerClient
       callback? 'No initial opened version was found for the currently active file'
 
   saveSecondaryFileAsDialog: (stringContent, extension, mimeType, callback) ->
-    data = { content: stringContent, extension, mimeType }
-    @_ui.saveSecondaryFileAsDialog data, (metadata) =>
-      # replace defaults
-      if extension
-        metadata.filename = CloudMetadata.newExtension metadata.filename, extension
-      if mimeType
-        metadata.mimeType = mimeType
-
-      @saveSecondaryFile stringContent, metadata, callback
+    if window.location.search.indexOf("saveSecondaryFileViaPostMessage") isnt -1
```

Comment:
Is there a reason to prefer URL query parameter here rather than using a property in the CFM config?